sql/opendata.sql

```diff
@@ -0,0 +1,137 @@
+--create tabular cpdb commitments table
+DROP TABLE IF EXISTS cpdb_opendata_commitments;
+WITH commit_proj as (
+SELECT 
+c.maprojid, 
+c.magency,
+c.projectid,
+p.description, 
+c.budgetline,
+c.plancommdate,
+c.commitmentdescription,
+c.commitmentcode,
+c.typc,
+c.typcname,
+c.ccnonexempt,
+c.ccexempt,
+p.citycost AS totalcityplannedcommit,
+c.nccstate,
+c.nccfederal, 
+c.nccother,
+p.noncitycost AS totalnoncityplannedcommit,
+p.totalcost AS totalplannedcommit,
+p.magencyacro, 
+p.magencyname, 
+c.ccpversion
+FROM cpdb_commitments as c,
+    cpdb_projects as p
+WHERE c.maprojid = p.maprojid)
+SELECT 
```

Comment:
Hey @caseysmithpgh do want to point out that the final opendata commitment table does not have a same order of columns as the data dictionary. Don't know if this needs to be addressed before final exporting.

Reply:
@td928 ah, yes good catch. I'm now noticing this is the case for the commitments table, Would this need to be corrected under the second select statement, scrapping c.*, and instead calling them in the correct order? Or is there a less lengthy approach you can think of?

Reply:
can you call the `alter_table` function and set the column order? https://stackoverflow.com/questions/285733/how-do-i-alter-the-position-of-a-column-in-a-postgresql-database-table#:~:text=PostgreSQL%20currently%20defines%20column%20order,you%20reach%20the%20desired%20layout.

Reply:
There might be a more slick way to do this though @AmandaDoyle @td928 might have some better ideas

Reply:
Order of the fields does matter.  You can call all the fields in one shot and do the join this way:
```
FROM cpdb.cpdb_commitments as c
LEFT JOIN cpdb.cpdb_projects as p ON c.maprojid = p.maprojid
LEFT JOIN cpdb.cpdb_budgets as b ON c.budgetline = b.budgetline AND c.maprojid = b.maprojid;
```